temba/flows/views.py

```diff
@@ -1260,6 +1448,84 @@ def get_gear_links(self):
                 links.append(dict(title=_("Previous Editor"), js_class="previous-editor", href="#"))
             return links
 
+    class PdfExportNext(AllowOnlyActiveFlowMixin, OrgObjPermsMixin, SmartReadView):
```

Comment:
I would like to see some inheritance here, something like reusing some classes changing just some parameters that could be changed using class attributes.